OnLoop/src/CoffeeOnLoop.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Scanner;
+
+public class CoffeeOnLoop {
+    static int coffeeBeans = 120;
```

Comment:
add an access modifier for this fields

Reply:
done! the variables are now declared as private